sample/build.gradle

```diff
@@ -57,6 +57,18 @@ android {
         exclude 'META-INF/gradle/incremental.annotation.processors'
     }
 }
+if (project.hasProperty("useKsp")) {
+    ksp {
```

Comment:
I think you might be able to take some inspiration from this PR. I have a mechanism to pass data to the annotation processor. You will need something similar for custom multi preview annotations @oas004

Reply:
Yes indeed! Thank you for this! This is very nice!😊😍 I think it is going to be very similar @vinaygaba  😊